vizro-ai/docs/pages/tutorials/explore-vizro-ai.md

```diff
@@ -57,12 +52,13 @@ Subsequently, we'll switch to German and prompt the visualization of life expect
 
     [GermanChart]: ../../assets/tutorials/chart/GermanExample.png
```

Comment:
@maxschulz-COL - "kreiere eine deutsche Visualisierung" - is this supposed to be like this? 😅  What does that change? Isn't that part redundant because it will create a visualisation in German anyways if the prompt is in German?

Reply:
Haha good question - @chiara-sophie was there a reason for this specific prompt?

Reply:
@maxschulz-COL - could you try running the example and see if we need it? Preferably I would cut that part from the docs as it sounds a bit weird :P Otherwise, I would rephrase to the suggested above?